### ISSUE TITLE
Add Cloud-Save-Syncing when launching via CLI/Steam-Shortcut

### ISSUE THREAD

**Issue by NoxDawnsong** [Heroic-Games-Launcher/HeroicGamesLauncher#2043]
### Problem description

It appears that, when launching via CLI/Steamshort, there is not cloud-save synchronization. 
After looking at the logs, there are no entries like 
```
INFO:    [Gog]:             Syncing saves for <appid>
```
when launching the game on the CLI via
```
flatpak run com.heroicgameslauncher.hgl --no-gui --no-sandbox "heroic://launch/<appid>"
```

I had a brief look at the source-code, and it appears as if `doAutoSyncSaves` in is never called when launching a game via `handleProtocol`, instead it only seems to be called by `handlePlay`, which in turn only seems to be attached to click-handlers. I might be wrong though, as mentioned i only browsed the source-code for a short time. 

### Feature description

Adding Cloud-Save-Syncing for CLI-launches will greatly help Steam-Deck players, that do want to utilize the game-mode features.

**Comment by Zarklord** [comment]
You are correct, additionally, if you launch from the library grid, it will also not sync saves, only by clicking the `Play Now` button will it actually sync your saves.

**Comment by redromnon** [comment]
Closing this issue since it should be fixed in v2.5.0-beta.3
